torchani/aev.py

```diff
@@ -121,30 +118,28 @@ def compute_shifts(cell, pbc, cutoff):
     ])
 
 
-# @torch.jit.script
 def neighbor_pairs(padding_mask, coordinates, cell, shifts, cutoff):
+    # type: (Tensor, Tensor, Tensor, Tensor, float) -> Tuple[Tensor, Tensor, Tensor]
     """Compute pairs of atoms that are neighbors
 
     Arguments:
-        padding_mask (:class:`torch.Tensor`): boolean tensor of shape
```

Comment:
Could you manually try if after this change, the cross link is still generated correctly? I am afraid that sphinx would trying to find `torchani.Tensor` instead of `torch.Tensor`.

Reply:
Resolved in 92584cc, sphinx was, in fact, trying to find `torchani.Tensor`